### PR TITLE
Add max_site_age envar

### LINKED ISSUE #752: Some valid channels are never enqueued for verification
We attempt to verify site channels every hour, however, if the channel is older than the `MAX_AGE` hardcoded in the `EnqueueSiteChannelVerifications` job, they will never be enqueued.
https://github.com/brave-intl/publishers/blob/b216b8d344bc74cc500372f2df8a3a68d325e9b5/app/jobs/enqueue_site_channel_verifications.rb#L3
If there is a bug that prevents an otherwise valid verification attempt from being processed for longer than `MAX_AGE`, these channels will never be verified.  This is precisely what happened with the recent bout of verification problems.

We should change `EnqueueSiteChannelVerifications` to depend on an environment variable instead of being hardcoded.  This way we can enqueue old (but valid) site verifications by adjusting the environment variable and not the code.

### PR DESCRIPTION
**Pull request by nvonpentz**
Resolves #752 

Open to other name suggestions!

Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave-intl/publishers/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Tagged reviewers.

Test Plan:

Reviewer Checklist:

Tests
- [ ] Adequate test coverage exists to prevent regressions

Security:
- [ ] No raw SQL -- Always prefer ActiveRecord query helpers ([more info on StackOverflow](https://stackoverflow.com/questions/41410752/rails-5-sql-injection#41452695))
- [ ] XSS is mitigated -- Avoid `html_safe` and `raw`; escape untrusted content from users and 3rd party APIs ([Rails XSS guide](https://brakemanpro.com/2017/09/08/cross-site-scripting-in-rails) and [OWASP XSS Prevention Cheat Sheet](https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet))

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -84,6 +85,7 @@ development:
   bat_twitter_url: "https://twitter.com/@attentiontoken"
   bat_reddit_url: "https://www.reddit.com/r/BATProject/"
   bat_rocketchat_url: "https://basicattentiontoken.rocket.chat/"
+  max_site_age: 6
```
Comment (lgebhardt) [inline config/secrets.yml]: Needs to be `6.weeks`

File: config/secrets.yml [hunk 2]
```diff
@@ -108,6 +110,7 @@ test:
...
```
Comment (lgebhardt) [inline config/secrets.yml]: Needs to be `6.weeks`

**Comment by nvonpentz** [comment]
@lgebhardt Thanks for taking a look.  I do specify `weeks` in when `MAX_AGE` is set in the job, but I could  instead specify it in the environment variable.  What do you think?

**Comment by lgebhardt** [comment]
@nvonpentz Sorry, I missed that. Ignore my comments - it's right the way it is.

**Comment by nvonpentz** [comment]
@lgebhardt I see no problem with being limited to weeks.

**Pull request merged**